superduperdb/components/graph.py

```diff
@@ -128,18 +172,32 @@ def __post_init__(self, artifacts):
             self.signature = 'singleton'
 
     def predict_one(self, *args):
+        '''
```

Comment:
Let us decide on a standard. Every sentence should have a `"."`.

Reply:
Added to all doc strings in components